src/core/ProductTile/ProductTile.stories.tsx

```diff
@@ -30,7 +30,27 @@ export const ProductTiles = {
     docs: {
       description: {
         story:
-          "Example usage: `<ProductTile name='pubsub' />`. Click on a tile to select it.",
+          "Example usage: `<ProductTile name='pubsub' selected={...} />`. Click on a tile to select it (`selected` must be controlled for selection to be enabled).",
+      },
+    },
+  },
+};
+
+export const StaticProductTiles = {
+  render: () => {
+    return (
+      <div className="grid sm:grid-cols-3 gap-32">
+        {Object.keys(products).map((product) => (
+          <ProductTile key={product} name={product as ProductName} />
```

Comment:
not important feedback, would be best to add more names in the key  to be more unique on each element in the list like this?

```
key={`${product}-static`}

key={`${product}-with-filled-icons`}
```


not sure if this matters as we only have the 1 doc page.

Reply:
Unless I'm misunderstanding you, I think this doesn't line up with the purpose of `key` here - which is to distinguish different items inside an array from each other. All of the keys of `products` are distinct, so there's no issue here, and adding a suffix like you describe wouldn't affect the outcome anyway

Reply:
If I'm right in what you were intending, keys don't apply globally - i.e. if two `map` functions in two different components in the same page use the same key value, then it wouldn't matter. It's locally scoped to that particular map.